apps/stocks/src/index.html

```diff
@@ -13,6 +13,10 @@
     />
   </head>
   <body>
-    <coding-challenge-root></coding-challenge-root>
+    <main role="main">
+      <h1 id="header" aria-label="Welcome to Stocks">Welcome to Stocks</h1>
```

Comment:
Page must contain a level-one heading and document must have one main landmark- Updated code for the same